common/perf-test-core/src/main/java/com/azure/perf/test/core/PerfStressProgram.java

```diff
@@ -18,26 +18,38 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.concurrent.Callable;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.ForkJoinPool;
+import java.util.concurrent.TimeUnit;
 import java.util.function.Supplier;
 import java.util.stream.IntStream;
-import java.util.stream.Stream;
 
 /**
  * Represents the main program class which reflectively runs and manages the performance tests.
  */
 public class PerfStressProgram {
     private static final int NANOSECONDS_PER_SECOND = 1_000_000_000;
 
-    private static int getCompletedOperations(PerfTestBase<?>[] tests) {
-        return Stream.of(tests).mapToInt(perfStressTest -> Long.valueOf(perfStressTest.getCompletedOperations()).intValue()).sum();
+    private static long getCompletedOperations(PerfTestBase<?>[] tests) {
+        long completedOperations = 0;
+        for (PerfTestBase<?> test : tests) {
+            completedOperations += test.getCompletedOperations();
+        }
+
+        return completedOperations;
     }
 
     private static double getOperationsPerSecond(PerfTestBase<?>[] tests) {
-        return IntStream.range(0, tests.length)
-            .mapToDouble(i -> tests[i].getCompletedOperations() / (((double) tests[i].lastCompletionNanoTime) / NANOSECONDS_PER_SECOND))
-            .sum();
```

Comment:
This had a fun issue where `Double.NaN` being added resulted in the value being `Double.NaN`.